### PR TITLE
Python plugin

### PR DESCRIPTION
**Pull request by gxb5443**
Adds support for compiling protobufs to Python.  Also adds enhancements to build script.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,21 +17,22 @@ RUN set -ex \
...
 	&& make install \
   && cd \
-	&& rm -rf /mnt/protobufs/ \
-	&& apk --no-cache add libstdc++ \
+	&& rm -rf /tmp/protobufs/ \
+  && rm -rf /tmp/protobufs.tar.gz \
+	&& apk --no-cache add libstdc++ \ 
 	&& apk del .pb-build \
-  && rm -rf /usr/local/include \
+  #&& rm -rf /usr/local/include \
```
Comment (poopoothegorilla) [inline Dockerfile]: do we want to just remove this? or write why it is commented?

File: python/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+MAINTAINER Core Services <coreservices@namely.com>
+
+ENV GRPC_RELEASE release-0_13
+
+# Install protoc python generator
+###################################
+
+RUN set -ex \
+	&& apk --no-cache --update add --virtual .python-builder \
+  make \
```
Comment (poopoothegorilla) [inline python/Dockerfile]: marked

**Comment by poopoothegorilla** [comment]
+1

File: Dockerfile (hunk 1 above)
Comment (bobbytables) [inline Dockerfile]: I concur.

**Pull request merged**